ts/src/vendor-specific-values.ts

```diff
@@ -24,6 +24,10 @@ export const Chromium: Readonly<VendorSpecificValues> = {
     [SourceType.event]: 6.5,
     [SourceType.navigation]: 11.5,
   },
+  // To accommodate our default configuration click=3,3,3 and view=1,1,1:
```

Comment:
Please move this comment to the PR description instead of this file.

Reply:
Also, please try to avoid the "click" and "view" terminology. The spec operates in terms of navigation sources and event sources. And the `3,3,3` syntax is not immediately understandable.